pages/answers/[messageId].js

```diff
@@ -52,18 +79,100 @@ const Answers = (props) => {
       externalNullifier,
       solidityProof
     }
-    console.log('body', body)
-    alert(`Submit question: ${answer}`)
+    console.log('ANSWERS PAGE | body', body)
 
     await axios.post('/api/postMessage', body, {
       timeout: API_REQUEST_TIMEOUT
     })
+
+    setSteps(['Generated zero knowledge proof', 'Submitted message transaction', 'Question successfully added'])
+
+    const updatedAnswers = [
+      {
+        id: Math.round(Math.random() * 100000000000).toString(),
```

Comment:
you can use a package here called uuid to be sure to get a unique identifier.

yarn add uuid

https://www.npmjs.com/package/uuid